client/me/help/help-contact/index.jsx

```diff
@@ -557,6 +553,24 @@ class HelpContact extends React.Component {
 		};
 	};
 
+	shouldShowHelpLanguagePrompt = ( variationSlug, currentUserLocale ) => {
+		if (
+			SUPPORT_HAPPYCHAT === variationSlug &&
+			config( 'livechat_support_locales' ).indexOf( currentUserLocale ) === -1
+		) {
+			return true;
+		}
+
+		if (
+			SUPPORT_UPWORK_TICKET === variationSlug &&
```

Comment:
There's three different "support variations" that lead to Tickets (though it's admittedly non-obvious):

- `SUPPORT_TICKET` is a standard user who has access to tickets but not chat
- `SUPPORT_CHAT_OVERFLOW` is when a user _should_ get a chat, but chat is full so they're "overflowing" to tickets
- `SUPPORT_UPWORK_TICKET` is for localized users on plans below Business, even if they would otherwise be chat-eligible

The variations are defined and set in [the `getInlineHelpSupportVariation` selector](https://github.com/Automattic/wp-calypso/blob/7655b6d1f326371990851da10ebb97f133fbdc17/client/state/selectors/get-inline-help-support-variation.js) and you can see that those three all lead to a Ticket from [this `switch` statement](https://github.com/Automattic/wp-calypso/blob/88d8adf8f05035da7230ae468426c6a197848264/client/me/help/help-contact/index.jsx#L447-L458).

So this line should check if `variationSlug` is any of those three (though I guess by definition, `SUPPORT_UPWORK_TICKET` should never happen if the user is outside an `upwork_support_locale` — so maybe we should just omit that variation from ever showing the warning?)